test/verify/check-networkmanager-firewall

```diff
@@ -64,6 +64,13 @@ class TestFirewall(NetworkCase):
         if m.image not in ["ubuntu-2004"]:
             m.execute("if virsh net-list --name | grep -q default; then until firewall-cmd --get-active-zones | grep -q libvirt; do sleep 1; done; fi")
 
+        # Temporary remove the test Dbus port 8765, this workaround can be
+        # removed once the pixel tests are merged and the image updated to not
+        # open port 8765
+        m.execute("firewall-cmd --permanent --remove-port=8765/tcp")
+        m.execute("firewall-cmd --remove-port=8765/tcp")
```

Comment:
Does this work if the port is already not opened? i.e. is it idempotent?

I'll trigger a rhel and debian test to make sure, and also trigger the mobile test run.

Reply:
--permanent does not seem to close the port when it's open. so we have to remove it explicitly